src/app/js/dataset/DefaultColumn.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import compose from 'recompose/compose';
+import withHandlers from 'recompose/withHandlers';
+import withState from 'recompose/withState';
+
+import { TableRowColumn } from 'material-ui/Table';
+import Format from '../formats/Format';
+import fetchByUri from '../lib/fetchByUri';
+import { getToken } from '../user';
+import { field as fieldPropTypes } from '../propTypes';
+
+export class DatasetColumnComponent extends Component {
+    componentWillMount() {
+        const { column, resource } = this.props;
+        const linkTransformer = column.transformers && column.transformers.find(t => t.operation === 'LINK');
+        if (linkTransformer) {
+            const uri = resource[linkTransformer.args.find(a => a.name === 'reference').value];
```

Comment:
it should not be the component that should make this call. But rather the saga that handle the change.